scripts/postprocess.py

```diff
@@ -190,9 +202,12 @@ def _calc_metrics(cols, df):
 #     "nodes":
 # }
 def _parse_line_based(fout, bench_name, nodes):
-    if "dlaf" in bench_name:
+    if bench_name.startswith("chol"):
         pstr_arr = []
-        pstr_res = "[{run_index:d}] {time:g}s {perf:g}GFlop/s ({matrix_rows:d}, {matrix_cols:d}) ({block_rows:d}, {block_cols:d}) ({grid_rows:d}, {grid_cols:d}) {:d}"
+        # Note that the optional fields must not have a space in front of them.
+        # Otherwise the space is required and parsing the optional field will
+        # fail.
+        pstr_res = "[{run_index:d}] {time:g}s {perf:g}GFlop/s{matrix_type:optional_text} ({matrix_rows:d}, {matrix_cols:d}) ({block_rows:d}, {block_cols:d}) ({grid_rows:d}, {grid_cols:d}) {:d}{backend:optional_text}"
```

Comment:
the `uplo` parameter after `matrix_type` is missing...isn't it?

Reply:
Not really, see: https://github.com/eth-cscs/DLA-Future/pull/443#discussion_r780154443.

"`matrix_type`" is now a combination the element type and possibly layout options in one string. I wasn't sure if `matrix_type` is descriptive enough but I figured the layout options could fit under that name. Does it make more sense with this explanation?

Reply:
Ah yeah, because there is no space in the output. Got it. thanks